agent/03_agent_deployment.sh

```diff
@@ -134,10 +208,40 @@ function attach_fleeting_iso() {
 
 write_pull_secret
 
-node0_ip=$(nth_ip $EXTERNAL_SUBNET_V4 20)
+# needed for assisted-service to run nmstatectl
+# This is temporary and will go away when https://github.com/nmstate/nmstate is used
+sudo yum install -y nmstate
+
+if [[ "$FLEETING_STATIC_IP_NODE0_ONLY" = "true" ]]; then
+    static_ips=1
+else
+    static_ips=$NUM_MASTERS+$NUM_WORKERS
+fi
+
+if [[ "$IP_STACK" = "v4" ]]; then
+    external_subnet=$EXTERNAL_SUBNET_V4
+else
+    external_subnet=$EXTERNAL_SUBNET_V6
+fi
+
+# Use an IP address outside of the range that is set for DHCP, just for sanity checks.
+# It shouldn't matter as the dhcp addresses will not be picked up when using static IPs
+base_ip=40
```

Comment:
Current dhcp range ends up to 60, just to avoid any potentital conflict it's better to stay out of it

Reply:
Additionally, see also here: https://github.com/metal3-io/metal3-dev-env/blob/681b4d26ab5f90b0d08679df36b82ebb74eca993/lib/network.sh#L133-L134

Reply:
Changed to start IPs at 80 to avoid any overlap with DHCP.